### PR TITLE
feat: update code api swagger

### PR DESCRIPTION
**Pull request by abhinav-harness**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- gitleaks: `trigger gitleaks`

### CODE REVIEW

File: harness/code/configuration.go [hunk 1]
```diff
@@ -56,34 +49,20 @@ type APIKey struct {
...
-	logger := logging.NewLogger()
-	if helpers.EnvVars.TfLog.Get() == "DEBUG" {
-		logger.SetLevel(log.DebugLevel)
-	}
-
 	cfg := &Configuration{
-		AccountId:     helpers.EnvVars.AccountId.Get(),
-		ApiKey:        helpers.EnvVars.PlatformApiKey.Get(),
-		BasePath:      helpers.EnvVars.Endpoint.GetWithDefault(utils.BaseUrl) + "/code/api/v1",
+		BasePath:      "http://app.harness.io/gateway/code/api/v1",
```
Comment (johannesHarness) [inline harness/code/configuration.go:62]: should basepath be hardcoded like this? In others the `code/api/v1` is set per operation.
(especially what if we have v1 v2 mixed at some point?)

File: harness/code/configuration.go [hunk 2]
```diff
@@ -56,34 +49,20 @@ type APIKey struct {
...
 		DefaultHeader: make(map[string]string),
-		HTTPClient:    utils.GetDefaultHttpClient(logger),
```
Comment (johannesHarness) [inline harness/code/configuration.go:82]: why are we removing httpclient default, logging, useragent, ...?
I guess accountID we anyway get from the methods (same for most of ng methods - why do they even have that in the client/config? :D)

Comment (johannesHarness) [reply harness/code/configuration.go:82]: oh I see they aren't being used by others really anyway

**Review by johannesHarness — APPROVED** [review]

File: harness/code/model_types_repository.go [hunk 3]
```diff
@@ -12,6 +12,7 @@ type TypesRepository struct {
 	Created        int32  `json:"created,omitempty"`
 	CreatedBy      int32  `json:"created_by,omitempty"`
 	DefaultBranch  string `json:"default_branch,omitempty"`
+	Deleted        int32  `json:"deleted,omitempty"`
```
Comment (atefehmohseni) [inline harness/code/model_types_repository.go:15]: Deleted is a pointer in Gitness, is there a reason you defined it as `int32` here?

Comment (johannesHarness) [reply harness/code/model_types_repository.go:15]: weird - seems to get lost in translation?!

**Review by johannesHarness — APPROVED** [review]

File: harness/code/model_types_repository.go (hunk 3 above)
Comment (atefehmohseni) [reply harness/code/model_types_repository.go:15]: yeah, it's `*int64` you have it `int32` not `*int32`

**Review by johannesHarness — APPROVED** [review]

**Pull request merged**